src/frontend/screens/MapScreen/index.tsx

```diff
@@ -104,8 +105,13 @@ export const MapScreen = () => {
         {coords && locationServicesEnabled && (
           <UserLocation minDisplacement={MIN_DISPLACEMENT} />
         )}
-        {isFinishedLoading && <ObservationMapLayer />}
-        {isFinishedLoading && <TrackPathLayer />}
+        {isFinishedLoading && (
+          <>
+            <ObservationMapLayer />
+            <TracksMapLayer />
+            <CurrentTrackMapLayer />
```

Comment:
should these be flipped? my understanding is that the existing tracks will cover the current track if implemented like this

Reply:
also, might be worth hiding behind the feature flag here

Reply:
https://github.com/digidem/comapeo-mobile/pull/408/commits/e66d44c1a0bf44df03042cdcf7c45ff8ec1d3331